roles/tas_single_node/tasks/podman/fulcio.yml

```diff
@@ -36,6 +36,23 @@
           {{ fulcio_config_content }}
   register: configmap_result
 
+- name: Create Fulcio Trusted OIDC CA Configmap
```

Comment:
Instead of creating multiple configmaps, would it be possible to have just a single one and use subPath? (E.g. see https://stackoverflow.com/a/72327498)

Reply:
No idea this existed, thanks!